.goreleaser.yml

```diff
@@ -24,7 +24,7 @@ builds:
       - -s -w -X github.com/stripe/stripe-cli/pkg/version.Version={{.Version}}
     binary: stripe
     env:
-      - CGO_ENABLED=1
+      - CGO_ENABLED=0
```

Comment:
Strictly speaking this was all that was needed to fix the issue. The rest is just opportunistic fixes/updates.